README.md

```diff
@@ -275,3 +278,12 @@ If you want to write an extension for Solidus, you can use the
 
 Solidus is an open source project and we encourage contributions. Please read
 [CONTRIBUTING.md](CONTRIBUTING.md) before contributing.
+
+
+## Sponsors
+
+Thank you to all our backers! 🙏 [Become a backer](https://opencollective.com/solidus#backer)
+
+Support this project by becoming a Key Stakeholder. Your logo will show up here with a link to your website. [Become a Key Stakeholder](https://opencollective.com/solidus#key-stakeholder)
+
+<img src="https://opencollective.com/solidus/tiers/key-stakeholder.svg">
```

Comment:
This image links to itself for some reason and this is not ideal since we are saying on the previous line:

> Your logo will show up here with a link to your website.

Reply:
Ah, so we may have to add those images manually. @seand7565 do you think you can do that for us? I can keep up with it as more people join that tier. 😄

Reply:
Yeah, absolutely. How should I deal with the image storage? Directly on the repo, like the Solidus logo? https://github.com/solidusio/solidus/blob/master/logo.svg

Or do we have another place to store these images?

Reply:
Can we grab them from OpenCollective profiles? For example `https://images.opencollective.com/proxy/images?src=https%3A%2F%2Flogo.clearbit.com%2Fenginecommerce.com`

Reply:
You got it! https://github.com/seand7565/solidus/tree/readme_oc#key-stakeholders